lib/moon_web/examples/design/table_example/stream.ex

```diff
@@ -0,0 +1,85 @@
+defmodule MoonWeb.Examples.Design.TableExample.Stream do
+  @moduledoc false
+
+  use Moon.LiveView
+  use MoonWeb, :example
+
+  alias Moon.Design.Table
+  alias Moon.Design.Table.Column
+  alias Moon.Components.Renderers.Datetime
+
+  def mount(_params, _session, socket) do
+    {:ok, stream(socket, :models, models())}
+  end
+
+  defp models() do
+    Enum.map(1..5, fn x ->
+      %{
+        id: x,
+        name: "Name #{x}",
+        created_at: DateTime.add(DateTime.utc_now(), -3600 + x)
+      }
+    end)
+  end
+
+  def render(assigns) do
+    ~F"""
+    <div>
+      <Table items={model <- @streams.models} selected={nil} body_attrs={%{"phx-update" => "stream"}}>
+        <Column name="id" label="ID">
+          {model.id}
+        </Column>
+        <Column name="name" label="First Name">
+          {model.name}
+        </Column>
+        <Column name="created_at" label="Created at">
+          <Datetime value={model.created_at} />
+        </Column>
+      </Table>
+    </div>
+    """
+  end
+
+  def code() do
+    """
+    use Phoenix.LiveView
+
+    alias Moon.Design.Table
+    alias Moon.Design.Table.Column
+    alias Moon.Components.Renderers.Datetime
+
+    def mount(_params, _session, socket) do
+      {:ok, stream(socket, :models, models())}
+    end
+
+    defp models() do
+      Enum.map(1..5, fn x ->
+        %{
+          id: x,
+          name: "Name \#{x}",
+          created_at: DateTime.add(DateTime.utc_now(), -3600 + x)
+        }
+      end)
+    end
+
+    def render(assigns) do
+      ~F\"""
+      <div>
+        <Table items={model <- @streams.models} selected={nil} body_attrs={%{"phx-update" => "stream"}}>
```

Comment:
hey just a question:
If I pass to `body_attrs` this : 
`body_attrs={%{"phx-viewport-bottom" => "{@page_info["hasNextPage"] && "next-page"}}"` 
the final result is going to be `phx-viewport-bottom={@page_info["hasNextPage"] && "next-page"}}` ?

Reply:
i believe it will be a syntax error))))))